deployment_helper/deployment_helper.py

```diff
@@ -237,6 +237,14 @@ def get_hosts(self):
                 _, path = host_input.split(':')
             with open(path) as fh:
                 new_hosts = [l.strip() for l in fh.readlines()]
+        elif '/' in host_input:     # any / should be a good enough indication this is a file name
```

Comment:
Just do:
`if host_input.lower().startswith('file:') or '/' in host_input:`
rather than copying and pasting the whole block.

Reply:
* your suggestion is cleaner. My python is rather useless so that seems fine.
* I hadn't considered the provided path being a directory and indeed I think this should be checked. That's also not checked now in your code.
* as for using os.path.pathsep again that makes sense. I'm not really familiar with all the python libraries and wasn't considering other OSes than UNIX-like systems.

Reply:
I think you misunderstood me. I was not saying that we should check that the given path is a directory or not. Consider the situation when there is a file called "host_list.txt" in the current directory. Since the file is in the current directory, the user would specify it without any path, and so the user input will not have a "/" in it. So, the code after your change, will assume that it is a host when actually it is a file.